promises.js

```diff
@@ -0,0 +1,103 @@
+async function getListOfBrands() {
```

Comment:
Lets start moving away from defining functions like this towards defining them using the arrow syntax.
```suggestion
const getListOfBrands = async ()  => {
```

Reply:
Good point. Changing it, Thank you